routes/main_routes.js

```diff
@@ -129,7 +132,21 @@ module.exports = function (app, passport) {
             res.redirect('/');
         });
 
-    app.use('/:language/tickets/', ticket_routes);
+    // =====================================
+    // JWT =================================
+    // =====================================
+    // JWT Login route
+    app.post("/jwt-login", function (req, res) {
+        passportLib.login(req.body.email, req.body.password, function(result, user, error) {
+            if (result && user) {
+                var token = passportLib.generateJwtToken(req.body.email);
```

Comment:
all logins should probably behave the same, no cookie, no service.
accessing directly to the login page is just a shortcut to avoid redirect.